src/.vuepress/util.js

```diff
@@ -145,16 +145,16 @@ function groupPagesByPath(pages) {
 }
 
 function sortPagesByOrderAndTitle(p1, p2) {
-  const o1 = get(p1, 'frontmatter.order', null);
-  const o2 = get(p2, 'frontmatter.order', null);
+  const o1 = +get(p1, 'frontmatter.order', NaN);
```

Comment:
I like the idea of forcing a better value than `null`, but I'm not sure that `NaN` is better. Actually, what happens here is that `order` is not defined, so we should force (and therefore check) to `undefined`, what's your opinion on this?

Reply:
I think `NaN` is better because like that function only deals with `number` type and the treatment is simpler

Reply:
You say it from a performance point of view?

Reply:
No, I think the code is cleaner when it only deals with `Number`

Reply:
:thinking: I actually agree, since the only valid value of `order` is a number. :raised_hands: